subsys/testsuite/coverage/coverage.c

```diff
@@ -247,6 +248,10 @@ void gcov_coverage_dump(void)
 
 		k_heap_free(&gcov_heap, buffer);
 		gcov_list = gcov_list->next;
+		if(gcov_list_first == gcov_list)
```

Comment:
Please fix style issues (spacing and brace placement):

    if (gcov_list_first == gcov_list) {
		goto coverage_dump_end;
    }